setup.py

```diff
@@ -25,9 +42,9 @@
     url="https://github.com/SymbiFlow/fasm",
     packages=setuptools.find_packages(exclude=('tests*',)),
     package_data={
-        'fasm': ['*.tx'],
+        'fasm': ['*.g4'],
+        'build': ['libparse_fasm.*']
```

Comment:
The output shared library probably end up inside the `fasm/` module directory structure?  I'm not confident on best practices here.

Reply:
I'm not sure either. I couldn't find any good examples using ctypes where the Python package distributes the library. It seems ctypes is usually used to wrap libraries installed separately.

Reply:
Try out what is shown here: https://stackoverflow.com/questions/42585210/extending-setuptools-extension-to-use-cmake-in-setup-py

Reply:
Thanks, I found that, and have updated setup.py. It now installs in `fasm/lib'.